server/routerlicious/packages/lambdas-driver/src/kafka-service/partitionManager.ts

```diff
@@ -58,6 +58,22 @@ export class PartitionManager extends EventEmitter {
 
 				this.emit("error", error, errorData);
 			});
+
+			this.consumer.on("checkpoint_success", (partitionId, queuedMessage) => {
+				Lumberjack.info(`Kafka checkpoint successful`, {
+					msgOffset: queuedMessage.offset,
+					topic: queuedMessage.topic,
+					msgPartition: queuedMessage.partition,
+				});
```

Comment:
Here again..we have logs during each checkpoint already in deli and scribe. This will only add one more log. However, it is true that it might be noisy for the other lambdas. Was that your concern? If that is the case, Kara maybe you can use the existing sampling logic: in addAlfredTrace->sampleMessages in Alfred so that we log this only once every 100 times

Reply:
@zhangxin511, these logs were also added at @pradeepvairamani's request for debugging purposes. To his point, we also have logs on each checkpoint. Do we want to remove or reduce the frequency of these as well?

Reply:
Sorry I don't know the frequency here, just thought we are logging for each checkpoint for all documents seems too much. Also I don't know how to correlate to the documentID, these are my concerns. I am good with suggestions from experts in checkpointing